### PR TITLE
[DCOS-59711] Added installation documentation for KUDO Spark

### PR DESCRIPTION
**Pull request by akirillov**
### What changes were proposed in this pull request?
Resolves [DCOS-59711: Document installation of KUDO Spark Operator](https://jira.mesosphere.com/browse/DCOS-59711)

This PR adds documentation for the basic installation steps for the beta release.

### Why are the changes needed?
The change is needed as a part of the operator documentation.

### How were the changes tested?
* tests from this repo
* by reviewing rendered markdown and verifying that links and the markup are correctly displayed

### CODE REVIEW

File: kudo-operator/docs/latest/installation.md [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
+
+Check out existing [limitations](./limitations.md) before installing a KUDO Spark instance. Currently, multi-instance 
+(multi-tenant) operator installation supports only a single instance per namespace. 
+
+
+```
+kubectl kudo install spark --instance=spark
+```
+
+Verify the if the deploy plan for `--instance=spark` is complete.
```
Comment (alembiewski) [inline kudo-operator/docs/latest/installation.md]:
```suggestion
Verify if the deploy plan for `--instance=spark` is complete.
```

File: kudo-operator/docs/latest/installation.md [hunk 2]
```diff
@@ -1,2 +1,61 @@
...
+  name: spark-operator-1
+  labels:
+    name: spark-operator-1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spark-operator-2
+  labels:
+    name: spark-operator-2
```
Comment (alembiewski) [inline kudo-operator/docs/latest/installation.md]:
```suggestion
    name: spark-operator-2
EOF
```

File: kudo-operator/docs/latest/installation.md (hunk 1 above)
Comment (akirillov) [reply kudo-operator/docs/latest/installation.md]: thanks for catching this. updated

File: kudo-operator/docs/latest/installation.md (hunk 2 above)
Comment (akirillov) [reply kudo-operator/docs/latest/installation.md]: good catch, thanks. updated

**Pull request merged**